### ISSUE TITLE
Mobile genetic elements from the maternal microbiome shape infant gut microbial assembly and metabolism

### ISSUE THREAD

**Issue by SvetlanaUP** [waldronlab/BugSigDBcuration#537]
Mobile genetic elements from the maternal microbiome shape infant gut microbial assembly and metabolism – Tommi Vatanen – Cell
https://www.cell.com/cell/fulltext/S0092-8674(22)01467-2

**Comment by OluchiTheAnalyst** [comment]
Good morning @SvetlanaUP 
I am Ndukauba Oluchi. I would love to curate this paper

Thank you so much.

**Comment by OluchiTheAnalyst** [comment]
Good morning @SvetlanaUP 
 I would like to delete **Signature 3** from **Experiment 1**  
Link : https://bugsigdb.org/w/index.php?title=Study_1276

**Comment by SvetlanaUP** [comment]
https://bugsigdb.org/Study_1276/Experiment_1/Signature_3 deleted.

**Comment by OluchiTheAnalyst** [comment]
I'd Like to submit this study for review
Link: https://bugsigdb.org/Study_1276

**Comment by AleruDivine** [comment]
Hi @OluchiTheAnalyst, well done on this curation attempt 👏
I have the following suggestions to improve the curation.

1. Group 0 for experiments 1 and 2 is gestational week 27 see Figure 2a caption "differences in species at delivery and 3 months postpartum compared to gestational week 27."
2. The significance threshold for these analyses was a false discovery rate (FDR)-corrected p value (q-value) of 0.25, this means that all the signatures in figure 2A are significant and should be curated accordingly.
3. The location of the Subjects is Finland.
4. A suitable condition for this is postpartum. Please note that "at delivery" is considered the beginning of the postpartum period.
5. The sample sizes are represented in Figure 1A (gestational week 27 = 69, delivery = 63, 3 months postpartum = 64)
6. The data transformation for MaAsLin2 is arscine square root.
7. These factors "antibiotic, breast feeding, delivery procedure, formula feeding" were considered as confounders in subsequent analyses of **infant metagenomic and metabolomic profiles** not the mothers.
8. Experiment 2 contrast groups are 3 months postpartum (Group 1) and gestational week 27 (group 0). 
9. All of the above also applies to experiment 2.
10. The signature in Figure 2D should also be added to experiment 1. It decreased for both experiments.
11. I'd like us to confirm if Figure S3C and S3E can be curated and how they should be curated.
12. After these corrections, the third experiment you have curated I believe should be deleted. Only 2 experiments are in figure 2A.
13. In Figure 5A, two experiments are shown. Infants were randomized into two groups: 40 (54%) to regular formula (Group 1) and 34 (46%) to hydrolyzed formula (Group 0). The data are stratified by intention-to-treat (left) and actual formula received (right). Please ensure this distinction is clearly indicated in the curation. Positive effect sizes represent increased signatures, while negative effect sizes represent decreased signatures. Results were obtained from general linear models (statistical test).

Looking forward to hearing from you😊

**Comment by OluchiTheAnalyst** [comment]
Thank you so much @AleruDivine for reviewing my first curation 😊 .Please I have a question **as regards to the the sample size**
_We analyzed longitudinal fecal and infant serum samples from 74 infants and 137 mothers, including 70 mother-infant dyads,_ as seen in the results section. so when I summed up the sample sizes (gestational week 27 = 69, delivery = 63, 3 months postpartum = 64), it was more than 137. I do not know if I am correct, but please can you help me confirm?

**Comment by AleruDivine** [comment]
Hello! @OluchiTheAnalyst
Yes it isn’t equal to 137 when you sum the numbers, but I don’t think it is supposed to be. 137 is the number of all the women they collected the samples from. But from figure 1A, there are individual sample sizes at every time point.

**Comment by OluchiTheAnalyst** [comment]
Hello @SvetlanaUP please could you Delete the whole of EXPERIMENT 3 for me
Link : https://bugsigdb.org/Study_1276

**Comment by SvetlanaUP** [comment]
Deleted.

**Comment by OluchiTheAnalyst** [comment]
Hi @AleruDivine _I'd like us to confirm if Figure S3C and S3E can be curated and how they should be curated._

**Comment by AleruDivine** [comment]
@OluchiTheAnalyst, yes I want us to confirm if the figures are curatable because of the captions, it doesn't directly say differential abundance, it is more like the association between bacterial signatures and the abundance of microbial glycoside hydrolases in infants and maternal samples (S3C) and associations of maternal B. cellulosilyticus with species in infants (S3E). I am very certain if they should or should not be curated that is why I suggested a confirmation.

As for the q values (q < 0.25), what ever is less than 0.25 is significant and 0.20 is within the significance threshold.

**Comment by OluchiTheAnalyst** [comment]
Okay @AleruDivine how do we confirm this?

**Comment by AleruDivine** [comment]
You can raise this concern during the office hours 😉

**Comment by OluchiTheAnalyst** [comment]
@AleruDivine Got it😌👌🏻

**Comment by OluchiTheAnalyst** [comment]
Hi @AleruDivine I have made all corrections
Link: https://bugsigdb.org/Study_1276

**Comment by AleruDivine** [comment]
Good morning @OluchiTheAnalyst, Thank you for the corrections so far.

I reviewed this and I still don't think the signatures in experiment 1 and 2 are curated accurately. All the signatures with the orange circles are increased for group 1 regardless of the size and all the signatures with blue circles are increased for group 2 regardless of their size. For example, only Dialister invisus and Bacteroides finegoldii are increased for Experiment 2. 

Red for increased and Blue for decreased.

I corrected sample sizes for experiments 5 and 6 please have a look at it. This is from the text above figure 5. I also updated the descriptions for the signatures. 

From experiment 3 - 6, I corrected the statistical tests and data transformation.

**Comment by OluchiTheAnalyst** [comment]
Oh! I am sorry for that oversight, I thought the signatures with negative values were for decreased that was why I made that grouping. I apologize for any inconvenience @AleruDivine

**Comment by OluchiTheAnalyst** [comment]
Hello @AleruDivine All corrections have been made. Please kindly review 😊

**Comment by AleruDivine** [comment]
Great job @OluchiTheAnalyst!

@SvetlanaUP, this review is complete.

**Comment by SvetlanaUP** [comment]
https://bugsigdb.org/Study_1276 reviewed.
